tests/integration/orders-update.spec.js

```diff
@@ -40,6 +43,7 @@ test('the module should modify an existing order', (t) => {
       ),
     )
     // Create needed data
+    .then(() => ordersUpdate.client.channels.ensure(channelKey, channelRole))
```

Comment:
Not sure what you mean, does this mean a change needs to be made or this PR is good to go?